AdyenUI/AdyenTheme.swift

```diff
@@ -36,6 +37,37 @@ public struct AdyenTheme {
     public init() {}
 }
 
+extension AdyenTheme: CheckoutTheme {
+
+    // Conformance to protocol
+
+    public var checkoutLabelStyle: CheckoutLabelStyle {
```

Comment:
Why do we need this get/set logic. Simply defining the property `var checkoutLabelStyle: CheckoutLabelStyle` should be enough. Not to mention there is optional casting which does not look good

Reply:
I have removed the get/set logic.